### PR TITLE
Update create_infra_env.adoc

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-6223
https://github.com/stolostron/rhacm-docs/pull/5325/files

### CODE REVIEW

**Comment by dockerymick** [comment]
Screenshot 2023-08-25 at 11 50 38 AM

**Review by cadawson — APPROVED** [review]
Looks good. Thanks!

**Comment by dockerymick** [comment]
@cadawson thanks for your approval Chris, I made one more change by moving the TOC after the list of features

File: clusters/cluster_lifecycle/create_infra_env.adoc [hunk 1]
```diff
@@ -20,6 +13,13 @@ Infrastructure environments support the following features:
 - Specify the infrastructure environment kernel arguments by editing the `InfraEnv` custom resource, which is required in some environments. See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/scalability_and_performance/clusters-at-the-network-far-edge#setting-managed-bare-metal-host-kernel-arguments_ztp-manual-install[Configuring the Discovery ISO kernel arguments for installations by using GitOps ZTP] for the instructions.
 - Specify your partition storage layout in a `custom-partitions.yaml` file when you deploy a bare metal cluster. See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/installing/index#configuring-storage-on-nodes_ipi-install-installation-workflow[Optional: Configuring storage on nodes] in the {ocp-short} documentation.
 
+* <<infra-env-prerequisites,Prerequisites>>
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/create_infra_env.adoc:18]:
```suggestion
Continue reading the documentation for details. 

* <<infra-env-prerequisites,Prerequisites>>
```

**Review by cadawson — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**